sdks/python/apache_beam/coders/stream.pyx

```diff
@@ -63,7 +63,9 @@ cdef class OutputStream(object):
         break
 
   cpdef write_bigendian_int64(self, libc.stdint.int64_t signed_v):
-    cdef libc.stdint.uint64_t v = signed_v
+    self.write_bigendian_uint64(signed_v)
```

Comment:
It is implicit as the method signature for `write_bigendian_uint64` has `libc.stdint.uint64_t`.